### PR TITLE
Restructure PopulationLib

### PR DESCRIPTION
**Pull request by mcwimm**
> **Working but benchmarks fail (i.e. output is different)**

Changes
- New structure without inheritance in PL
- Two sections 
   - Population, PlantGroup, Plant: manages plant attributes
   - Dispersal: manages where new plants are planted, and what initial attributes they have
- Plant module is now initialized at the group level (not plant as before)
- Parameters to describe the initial plant are more flexible but needs to be tested

ToDos
- [x] Fehlersuche 
- [x] random anpassen 
- [x] update benchmarks
- [x] test with kiwi initial populations
- documentation - in separate PR
- [ ] remove todos within code
- [x] update other xml files
- [ ] conceptional problem with two plant models - issue #306

### CODE REVIEW

File: PopulationLib/Dispersal/FromFile/FromFile.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                exit()
+        positions = {"x": plant_attributes_file["x"].flatten(),
+                     "y": plant_attributes_file["y"].flatten()}
+        return positions, geometry
+
+    def getGeometryList(self):
+        # ToDo: Liste mit notwendigen Parametern irgendwoher holen?
+        plant_model = self.xml_args.find("vegetation_model_type").text
+        if plant_model == "Default":
+            # ToDo: Welche Geomtrie soll für Default definiert werden? Anpassung Benchmarks&ini_pop.csv notwendig
```
Comment (jvollhueter) [inline PopulationLib/Dispersal/FromFile/FromFile.py:37]: After merging I will implement this in a new PR

**Review by jvollhueter — APPROVED** [review]
could not find any problems, let's merge it!

**Review by ChrisWudel — APPROVED** [review]
I haven't noticed any problems either, should be working now.

**Review by jvollhueter — APPROVED** [review]
So let's merge it!

**Pull request merged**